faros_event.sh

```diff
@@ -360,6 +367,15 @@ function resolveDeploymentInput() {
     deployment_status_details=${deploymant_status_details:-$FAROS_DEPLOYMENT_STATUS_DETAILS}
     deployment_start_time=${deployment_start_time:-$FAROS_DEPLOYMENT_START_TIME}
     deployment_end_time=${deployment_end_time:-$FAROS_DEPLOYMENT_END_TIME}
+
+    if ! [[ ${ENVS[*]} =~ (^|[[:space:]])"$deployment_env"($|[[:space:]]) ]] ; then
+      echo "Invalid deployment environment $deployment_env. Allowed values: ${envs}";
+      exit 1
```

Comment:
If you want I have those two helper functions `err` and `fail` which tack on the time and the level. For example:
```sh
if ! [[ ${ENVS[*]} =~ (^|[[:space:]])"$deployment_env"($|[[:space:]]) ]] ; then
      err "Invalid deployment environment $deployment_env. Allowed values: ${envs}";
      fail
fi
```
Which results in the following output:
<img width="826" alt="Screen Shot 2021-07-16 at 8 18 04 PM" src="https://user-images.githubusercontent.com/10718300/126023814-fe53d258-fbfa-4626-81bd-5f1409827891.png">

Reply:
Cool! Updated.